e2e/containers.e2e.spec.ts

```diff
@@ -0,0 +1,106 @@
+import os from 'os';
+import path from 'path';
+
+import { test, expect } from '@playwright/test';
+import { ElectronApplication, BrowserContext, _electron, Page } from 'playwright';
+
+import { NavPage } from './pages/nav-page';
+import { createDefaultSettings, packageLogs, reportAsset, tool } from './utils/TestUtils';
+
+import { ContainerEngine, Settings } from '@/config/settings';
+import { RecursivePartial } from '~/utils/typeUtils';
+
+let page: Page;
+
+/**
+ * Using test.describe.serial make the test execute step by step, as described on each `test()` order
+ * Playwright executes test in parallel by default and it will not work for our app backend loading process.
+ * */
+test.describe.serial('Containers Test', () => {
```

Comment:
Perhaps `test.descibe.serial('container engine', …`? (That is, the section is describing the container engine, not the test.)